### PR TITLE
More formatting examples, small rewordings

### PR DESCRIPTION
**Pull request by joethei**
~ Adding callout examples as per <https://forum.obsidian.md/t/add-callout-examples-with-colours-to-the-online-help/40725>
~ closing #266 by making the wording clearer and adding examples
~ closing #267 as the changes there are included in this PR
+ More examples

Documenting some syntax that leads to questions in discord sometimes
+ alternative heading syntax
+ loose lists

### CODE REVIEW

**Review by marcusolsson — CHANGES_REQUESTED** [review]

File: en/How to/Format your notes.md [hunk 1]
```diff
@@ -368,16 +441,27 @@ $$\begin{vmatrix}a & b\\
...
+---
+
+```md
+You can also do inline math like $e^{2i\pi} = 1$ .
+```
+
 You can also do inline math like $e^{2i\pi} = 1$ .
 
-Obsidian uses [Mathjax](http://docs.mathjax.org/en/latest/basic/mathjax.html). You can check which packages are supported in Mathjax [here](http://docs.mathjax.org/en/latest/input/tex/extensions/index.html).
+To render math from LaTeX notion Obsidian uses [MathJax](http://docs.mathjax.org/en/latest/basic/mathjax.html).
```
[the remark this answers is not in the document]
Comment (joethei) [reply en/How to/Format your notes.md:454]: typo: should be notation

**Comment by ericaxu** [comment]
There seems to be some conflicts. Let me know when it's ready to merge @joethei @marcusolsson

File: en/How to/Use callouts.md [hunk 2]
```diff
@@ -70,3 +199,11 @@ Snippets and plugins can define custom callouts, too, or overwrite the default o
     --callout-icon: '<svg>...custom svg...</svg>';
 }
 ```
+
+### Developer Notes
+
+The syntax has been chosen because:
```
Comment (marcusolsson) [inline en/How to/Use callouts.md:205]:
```suggestion
We choose the syntax because:

```

File: en/How to/Use callouts.md [hunk 3]
```diff
@@ -70,3 +199,11 @@ Snippets and plugins can define custom callouts, too, or overwrite the default o
...
+- it gracefully falls back to blockquotes in unsuported markdown renderers
```
Comment (marcusolsson) [inline en/How to/Use callouts.md:206]:
```suggestion
- it falls back to blockquotes in unsupported markdown renderers
```

File: en/How to/Use callouts.md [hunk 4]
```diff
@@ -70,3 +199,11 @@ Snippets and plugins can define custom callouts, too, or overwrite the default o
...
+- it parses all markdown, including links, embeds, etc.
```
Comment (marcusolsson) [inline en/How to/Use callouts.md:207]:
```suggestion
- it parses all Markdown, including links, embeds, etc.
```

File: en/How to/Use callouts.md [hunk 5]
```diff
@@ -70,3 +199,11 @@ Snippets and plugins can define custom callouts, too, or overwrite the default o
...
+- [[Internal link|Internal link's]] are recognized and will show up in [[Backlinks]], [[Outgoing links]] and [[Graph view]]
```
Comment (marcusolsson) [inline en/How to/Use callouts.md:208]:
```suggestion
- it recognizes [[Internal link|Internal link's]] and will shows up in [[Backlinks]], [[Outgoing links]], and [[Graph view]]
```

**Review by marcusolsson — APPROVED** [review]

**Comment by ericaxu** [comment]
Thanks all! ❤️

**Pull request merged**